monkestation/code/modules/hydroponics/grown/coconut.dm

```diff
@@ -0,0 +1,48 @@
+/obj/item/food/grown/coconut
+	name = "Coconut"
+	desc = "a fruit with a hardshell, filled with sweet coconut milk"
+
+	icon = 'monkestation/icons/obj/hydroponics/fruit.dmi'
+	icon_state = "coconut"
+
+	bite_consumption_mod = 3
+	juice_results = list(/datum/reagent/consumable/milk = 0)
+	seed = /obj/item/seeds/coconut
+
+/obj/item/food/grown/shell/coconut_gun
+	seed = /obj/item/seeds/coconut/gun
+	name = "coconut gun"
+	desc = "It smells like burning."
```

Comment:
is this meant to not make grammatical sense?

Reply:
i believe this is just the gatfruits desc so yea